<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sondage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabprincipal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Personne</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>541</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_warning_2">
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_age_2">
           <property name="text">
            <string>âge : </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="txtbox_age_2"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_sexe_2">
           <property name="text">
            <string>sexe : </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="rdb_homme_2">
             <property name="text">
              <string>homme</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rdb_femme_2">
             <property name="text">
              <string>femme</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_etat_civil_2">
           <property name="text">
            <string>état civil : </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="cb_etat_civil_2">
           <item>
            <property name="text">
             <string>marié</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>veuf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>célibataire</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>vie_maritale</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbl_compo_foyer_2">
           <property name="text">
            <string>composition du foyer : </string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QFormLayout" name="formLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_nb_personnes_2">
             <property name="text">
              <string>nombre de personnes : </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spbox_nb_personnes_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_enfants_2">
             <property name="text">
              <string>dont enfants : </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spbox_enfants_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_animaux_2">
             <property name="text">
              <string>dont animaux domestiques : </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="spbox_animaux_2"/>
           </item>
           <item row="0" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbl_revenu_2">
           <property name="text">
            <string>tranche de revenu : </string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="cb_revenu_2">
           <item>
            <property name="text">
             <string>de 10 000 € à 25 000 €</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>de 25 000 € à 70 000 €</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>de 70 000 € à 150 000 €</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>+ de 150 000 €</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Au Bon Lait</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>551</width>
        <height>474</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="quelleSMarqueSDeYaourtsConnaissezVousLabel_2">
           <property name="maximumSize">
            <size>
             <width>16776500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Quelle(s) marque(s)  de yaourts connaissez-vous?</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Quel(s) type(s) de yaourts consommez-vous?</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>A quelle fréquence achetez-vous des yaourts? </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_frequence">
             <item>
              <property name="text">
               <string>jamais</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>au moins trois fois par semaine</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>au moins une fois par semaine</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>au moins une fois par mois</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <layout class="QGridLayout" name="grid_types">
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_19">
             <property name="text">
              <string>mixés</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_20">
             <property name="text">
              <string>fruits avec morceaux</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="checkBox_21">
             <property name="text">
              <string>brassés</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="checkBox_22">
             <property name="text">
              <string>bifidus fruits</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="checkBox_23">
             <property name="text">
              <string>bifidus  nature</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>allégé</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>à boire</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>anti-cholestérol</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_24">
             <property name="text">
              <string>nature</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Quelle(s) marque(s) avez-vous achetée(s) au cours des 6 derniers mois?</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QListView" name="listview_marques_connues">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QListView" name="listview_marques_achetees">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveauSonde"/>
    <addaction name="separator"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuSondage">
    <property name="title">
     <string>Sondage</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNouveauSondage"/>
    <addaction name="separator"/>
    <addaction name="actionAfficher_historique"/>
   </widget>
   <widget class="QMenu" name="menuStatistiques">
    <property name="title">
     <string>Statistiques</string>
    </property>
    <addaction name="actionG_n_rer_statistiques"/>
   </widget>
   <widget class="QMenu" name="menuSond_s">
    <property name="title">
     <string>Sondés</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="separator"/>
    <addaction name="actionGestion"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuSondage"/>
   <addaction name="menuStatistiques"/>
   <addaction name="menuSond_s"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveauSonde">
   <property name="text">
    <string>Nouveau sondé</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir sondé</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionNouveauSondage">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionAfficher_historique">
   <property name="text">
    <string>Afficher historique</string>
   </property>
  </action>
  <action name="actionG_n_rer_statistiques">
   <property name="text">
    <string>Générer statistiques</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionGestion">
   <property name="text">
    <string>Gestion</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
